rules/linux-kernel.mk

```diff
@@ -7,8 +7,19 @@ KERNEL_SUBVERSION = 1+deb9u5
 
 export KVERSION_SHORT KVERSION KERNEL_VERSION KERNEL_SUBVERSION
 
+
+SMPATH=$(SRC_PATH)/sonic-linux-kernel/
+SM_DEP_LIST  := Makefile
```

Comment:
what does this SM stand for?

Reply:
SM stands for Sub Module in the sonic repo
Initially, the DPKG caching framework is supported only for the Linux submodule, later, we have enhanced the DPKG caching framework to cover all the sub modules and Debian packages in the sonic repro. 
We will close this one and will raise a new pull request for the enhanced version of the DPKG caching framework.